assets/sass/main.min.scss

```diff
@@ -27,3 +27,4 @@
 @import 'pages/_who.scss';
 @import 'pages/_roadmap.scss';
 @import 'pages/_case-study.scss';
+@import 'pages/_docs.scss';
```

Comment:
This still needs to be moved to `docs.min.scss` or it will not be used in the docs views.